bot/cogs/nasa.py

```diff
@@ -0,0 +1,167 @@
+import os
+import random
+
+import aiohttp
+
+from discord import Color, Embed
+from discord.ext.commands import (
+    Bot,
+    BucketType,
+    Cog,
+    Context,
+    command,
+    cooldown
+)
+
+NASA_API = os.getenv("NASA_API")
+
+
+def remove_tags(text: str) -> None:
+    import re
+    clean = re.compile('<.*?>')
+    return re.sub(clean, '', text)
+
+
+class Nasa(Cog):
+    def __init__(self, bot: Bot) -> None:
+        self.bot = bot
+        self.session = aiohttp.ClientSession()
```

Comment:
Make sure `NASA_API` isn't `None`, if it is, unload the cog by calling `self.cog_unload()` as I've done in #83 like this:
https://github.com/The-Codin-Hole/HotWired-Bot/blob/9483896154a45903db49e4e3bf6e1743ec82e33b/bot/cogs/reddit.py#L54-L63

While you won't be able to use logging to signify that the cog didn't load, it will prevent the errors and I will add the logging from my PR.